go/libraries/doltcore/table/typed/parquet/writer.go

```diff
@@ -114,10 +114,13 @@ func (pwr *ParquetWriter) WriteSqlRow(ctx context.Context, r sql.Row) error {
 			// convert datetime and time types to int64
 			switch colT.TypeInfo.GetTypeIdentifier() {
 			case typeinfo.DatetimeTypeIdentifier:
-				val = val.(time.Time).Unix()
+				val = val.(time.Time).UnixMicro()
 				sqlType = sql.Int64
 			case typeinfo.TimeTypeIdentifier:
-				val = int64(val.(sql.Timespan))
+				v := int64(val.(sql.Timespan))
+				// take abs(val) TODO : negative timespan goes out of range when imported into pandas
+				//shift := v >> 63
+				val = v //(v ^ shift) - shift
```

Comment:
Looks like this logic hasn't changed at all? I couldn't figure out what the comments were for. Is this still work in progress?

Reply:
oops, I was doing more testing on negative value. I added abs(v) function to avoid negative value causing panic for now.